spec/src/main/asciidoc/SOAPAttachments.adoc

```diff
@@ -47,14 +38,17 @@ The proposed changes specified in the fourth
 JSR067 changelog have been incorporated into this document as SAAJ
 Specification, version 1.4.
 
+Starting with Jakarta SOAP Attachments Specification, Version 2.0, 
+this document is developed under https://jakarta.ee/about/jesp/[Jakarta EE Specification Process]
+
 === 
 
 image:soapatt-5.png[image]
 
 Acknowledgements
 
 This maintenance release is the product of
-collaborative work within the Java community.
+collaborative work within the Jakarta EE Working Group.
```

Comment:
I do not think this change is required - Jakarta EE Working Group is supposed to be subset of Java community - at least that is how I understand it.

Reply:
I could probably accept this being reworded to sth like: "This release is the product of collaborative work within the Jakarta EE community."